### PR TITLE
[docs-only] Clarify the license of ownCloud Infinite Scale binary packages.

### PR DESCRIPTION
**Pull request by dragotin**
Clarification of the license of Infinite Scale, especially the stable binary builds under EULA

### CODE REVIEW

**Comment by micbar** [comment]
@dragotin there is already a section about the EULA in the readme. https://github.com/owncloud/ocis/blob/master/README.md?plain=1#L126

**Comment by dragotin** [comment]
Thanks, updated

File: docs/ocis/license.md [hunk 1]
```diff
@@ -7,4 +7,8 @@ geekdocEditPath: edit/master/docs/ocis
 geekdocFilePath: license.md
 ---
 
-This project is licensed under the [Apache 2.0](https://github.com/owncloud/ocis/blob/master/LICENSE) license. For the license of the used libraries you have to check the respective sources.
+The source code of the project is licensed under the [Apache 2.0](https://github.com/owncloud/ocis/blob/master/LICENSE) license. For the license of the used libraries you have to check the respective sources.
+
+Stable, supported binary builds of ownCloud Infinte Scale that will be distributed by ownCloud GmbH are covered by a non OSS Freemium License [EULA](https://owncloud.com/license-owncloud-infinite-scale/). This protects additional efforts that ownCloud GmbH is putting into these builds.
```
Comment (phil-davis) [inline docs/ocis/license.md:12]:
```suggestion
Stable, supported binary builds of ownCloud Infinite Scale that will be distributed by ownCloud GmbH are covered by a non OSS Freemium License [EULA](https://owncloud.com/license-owncloud-infinite-scale/). This protects additional efforts that ownCloud GmbH is putting into these builds.
```

**Review by micbar — APPROVED** [review]

**Review by phil-davis — APPROVED** [review]
Text looks OK

**Pull request merged**